VERSION

```diff
@@ -1 +1 @@
-0.9.0
+0.9.1
```

Comment:
for using (fluent-pluigin-webhdfs)[https://github.com/fluent/fluent-plugin-webhdfs/pull/83] pr,
may i have to miner version up?
0.9.0 -> 0.9.1 (x)
0.9.0 -> 0.10.0

how can i bound the version in fluent-pluigin-webhdfs?

Reply:
I'll bump the minor version for this release.